docs/3_finetuner_in_action/3_3_text_to_image.md

```diff
@@ -1,37 +1,39 @@
 (text-to-image)=
-# Text to image search using CLIP
+# CLIP for text to image search
 
 This guide will showcase fine-tuning a `CLIP` model for text to image retrieval.
 
-## Task overview
-We'll be fine-tuning CLIP on a fashion captioning dataset which contains information about fashion products.
+## Task
+We'll be fine-tuning CLIP on the [fashion captioning dataset](https://github.com/xuewyang/Fashion_Captioning) which contains information about fashion products.
 
-For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two chunks: an image document and a text document holding the description of the product.
+For each product the dataset contains a title and images of multiple variants of the product. We constructed a parent `Document` for each picture, which contains two [chunks](https://docarray.jina.ai/fundamentals/document/nested/#nested-structure): an image document and a text document holding the description of the product.
```

Comment:
Maybe `Document` can still receive a developer reference here? It is its first mention in this guide.